### ISSUE TITLE
SALT2mu.exe: Combining varname_gamma and CUTWIN

### ISSUE THREAD

**Issue by wycombe7** [RickKessler/SNANA#115]
Hi Rick, 
I'm running SALT2mu.exe with alternative columns for gamma instead of HOST_LOGMASS. As such, I have instigated a MUOPT with the following entry: 
`MUOPT:  opt_biascor=2   varname_gamma='LOCAL_UR'`
replacing "HOST_LOGMASS" with "LOCAL_UR". This syntax works successfully and SALT2mu runs to conclusion. 

However, to test the sensitive of my results, I also want to cut on this variable. To do this, I had planned on using the CUTWIN function concurrently with varname_gamma. However, this doesn't seem to work; e.g.: 
`MUOPT:  opt_biascor=2   varname_gamma='LOCAL_UR'   CUTWIN LOCAL_UR 0.90 777`

results in a ABORT, stating that 'LOCAL_UR' is an invalid CUTWIN variable. This is a little weird, as, it seems that the log for the to query above lists LOCAL_UR as a viable CUTWIN variable in the LOG. 

Any thoughts on how to proceed? In short, I'd like to use the `varname_gamma` method to replace HOST_LOGMASS and then cut on it in a similar fashion to `CUTWIN HOST_LOGMASS 10 13`. 

As tests, I've tried replacing `CUTWIN LOCAL_UR` with `CUTWIN LOG_HOSTMASS` in my proposed MUOPT, but this leads to an abort. I have also tried using logmass_min/logmass_max keywords, but these cut the BIASCOR and as my parameter is not defined there this fails too. 

An example can be found: `/project2/rkessler/SURVEYS/DES/USERS/smithmat/SNANA_Data/DES3YR/SALT2/DES_v8/DES_only/SALT2mu_SALT2fits_local/SALT2mu_DESonly_localenv.input` 

In short - how can I cut in SALT2mu on a parameter input through the `varname_gamma` method?

Best wishes,

Mat

**Comment by RickKessler** [comment]
Problem is that it fails on the sim because LOCAL_UR does not exist for the sim. However, looks like this issue was solved back in October 2018. Can request DATAONLY cut with
CUTWIN(DATAONLY) LOCAL_UR 0.90 777
